Doc/library/typing.rst

```diff
@@ -988,13 +988,6 @@ The module defines the following classes, functions and decorators:
 
    ``Optional[X]`` is equivalent to ``Union[X, None]``.
 
-   Note that this is not the same concept as an optional argument,
-   which is one that has a default.  An optional argument with a
```

Comment:
Please keep the first sentence (i.e. `Note that this is not the same concept as an optional argument, which is one that has a default.`). You might also add examples, e.g. `def (arg: int = 0)` vs. `def foo(a: Optional[int] = None)`.